supporting_files/species_EnsemblPlants.txt.default

```diff
@@ -0,0 +1,115 @@
+#name	species	division	taxonomy_id	assembly	assembly_accession	genebuild	variation	microarray	pan_compara	peptide_compara	genome_alignments	other_alignments	core_db	species_id
```

Comment:
Is this file placed here for testing/satisfy installation or we expect to update this file here for actual usage? I presume this would need to be updated on every E! release, and if that is the case then maybe we need to consider a different place for this.